.circleci/config.yml

```diff
@@ -94,16 +128,26 @@ jobs:
                 echo "Skipping integration tests"
             fi
           no_output_timeout: 3000s
-      - run:
-          name: Combine Coverage Results
-          working_directory: ~/curation
-          command: |
-              . tests/combine_coverage.sh
-          no_output_timeout: 30s
-      # Save test results
-      - store_test_results:
-          path: ~/curation/tests/results/junit
-      # Save artifacts
-      - store_artifacts:
-          path: ~/curation/tests/results/coverage
-          destination: test_results
+      - test_teardown
+
+workflows:
+  version: 2
+  test:
+    jobs:
+      - unit_test
```

Comment:
Should this be `- unit_test:`??  Unit tests didn't run on this PR.

Reply:
They ran:

![image](https://user-images.githubusercontent.com/822298/74556556-27dd2a00-4f13-11ea-8456-7356a376061e.png)


Separately however, they did fail. It looks like some of these tests depend on ADC being configured since I am no longer authenticating the unit test process. I will fix those today - but if you're able please review the config in the meantime.

Reply:
Also, this is a nice side-effect of not setting up buckets or auth on unit tests, it forcibly reveals the non-hermetic dependencies.

Reply:
You're right.  I skimmed and did not notice that the unit tests and integration tests were reporting separately now.  I like revealing the dependencies.  These shouldn't be needed for unit testing anyway.  Thanks CH.